src/lib/components/ui/show-more/show-more.svelte

```diff
@@ -3,12 +3,17 @@
 	import { Button } from '$lib/components/ui/button';
 	import { cn } from '$lib/utils';
 
-	export let expanded: boolean;
+	type Props = {
+		expanded: boolean;
+		onclick: () => void;
+	};
+
+	let { expanded, onclick }: Props = $props();
 </script>
 
 <div class="relative flex min-h-[30px] w-[calc(100%-40px)] items-center p-5">
 	<div aria-hidden="true" class="h-px w-full -translate-x-px bg-gray-alpha-400"></div>
-	<Button variant="secondary" size="sm" shape="rounded" aria-label="Show less" on:click>
+	<Button variant="secondary" size="sm" shape="rounded" aria-label="Show less" {onclick}>
 		Show less
 		<Icons.ChevronLeft
 			class={cn('size-4 -rotate-90 transition-transform duration-200 ease-in-out', {
```

Comment:
This is a bit oversized compared to the original maybe `size-3` would work better.